packages/design/components/Toast/types.ts

```diff
@@ -1,5 +1,8 @@
+import type { MessageType } from '@bangumi/design';
```

Comment:
```suggestion
import type { MessageType } from '../Message';
```
有两处也像引入组件一样引入 MessageType 吧，虽然现在没什么问题，还是不要循环引用了

Reply:
好，一开始就是直接引用的，后来参考了别处反而改成了这种循环引用，改一下好了